step-templates/ansible-run-playbook.json

```diff
@@ -0,0 +1,34 @@
+{
+  "Id": "f0af011b-f99d-4809-84fe-af005b619fa2",
+  "Name": "Run Ansible Playbook",
+  "Description": "Step template to run an ansible playbook using bash",
```

Comment:
Can you expand the description to include the pre-requisites for this step to run successfully, Based on the logic in the script, it needs the tool `ansible-playbook`, but does it need anything else?

For example here is what we have for the [AZ key vault retrieve secrets step](https://library.octopus.com/step-templates/6f59f8aa-b2db-4f7a-b02d-a72c13d386f0/actiontemplate-azure-key-vault-retrieve-secrets)

![image](https://user-images.githubusercontent.com/1418993/219351499-3efdf977-5520-4d15-9011-d5f0ce04b064.png)